tests/Behat/Context/Reorder/ReorderContext.php

```diff
@@ -5,16 +5,45 @@
 namespace Tests\Sylius\CustomerReorderPlugin\Behat\Context\Reorder;
 
 use Behat\Behat\Context\Context;
+use Behat\Mink\Session;
+use function GuzzleHttp\Psr7\str;
 use Sylius\Component\Core\Model\AddressInterface;
 
 final class ReorderContext implements Context
 {
+    /** @var Session */
+    private $session;
+
+    public function __construct(Session $session)
+    {
+        $this->session = $session;
+    }
+
     /**
      * @Then I should see reorder button next to the order :orderNumber
      */
     public function iShouldSeeReorderButtonNextToTheOrder(string $orderNumber): void
     {
-
+        $orderHistory = $this->session->getPage()->findAll('css', 'td');
+
+        $isOrderPresent = false;
+        $isReorderButtonVisible = false;
+
+        foreach ($orderHistory as $orderData) {
+            if (strpos($orderData->getText(), $orderNumber) != false) {
+                $isOrderPresent = true;
+            }
+
+            if (strpos($orderData->getText(), 'Reorder') != false) {
+                $isReorderButtonVisible = true;
```

Comment:
Why can't we throw individual exceptions inside these `if` statements? It would prevent us from setting some artificial variables just to check them later... We can throw in this one `\Exception(sprintf('There is no reorder button next to order %s', $orderNumber));` and `\Exception(sprintf('There is order %s', $orderNumber));` or sth similar above.

Reply:
I meant, that we can get rid of these variables `$isOrderPresent` and `$isReorderButtonVisible` and throw exceptions inside **this** conditions :)